client-side/src/axios/middleware.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import dotenv from 'dotenv'
+
+dotenv.config()

```

Comment:
please add ; to this line and the next one too

Reply:
✅